docs/ref/contrib/auth.txt

```diff
@@ -315,6 +315,14 @@ Manager methods
         Same as :meth:`create_user`, but sets :attr:`~models.User.is_staff` and
         :attr:`~models.User.is_superuser` to ``True``.
 
+    .. method:: with_perm(perm, obj=None)
+
+        .. versionadded:: 1.11
+
+        Returns a queryset containing :class:`~django.contrib.auth.models.User`
+        objects that have the given permission ``perm`` in the form of
+        ``"<app label>.<permission codename>"``, including superusers.
```

Comment:
What about documenting the fact that it only checks the first backend that implements `with_perm()`? The rationale behind that decision could be interesting. What about the need to specify which backends to check?